innovation.game.php

```diff
@@ -1732,6 +1829,12 @@ function notifyWithTwoPlayersInvolved($card, $transferInfo, $progressInfo) {
                 $message_for_opponent = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from ${your} hand to his hand.');
                 $message_for_others = clienttranslate('${player_name} transfers a ${<}${age}${>} from ${opponent_name}\'s hand to his hand.');
                 break;
+
+            case 'hand->achievements':
+                $message_for_player = clienttranslate('${You} transfer ${<}${age}${>} ${<<}${name}${>>} from ${opponent_name}\'s hand to your achievements.');
```

Comment:
${You} transfer a ${<}${age}${>} from ${opponent_name}\'s hand to your achievements.